src/views/open-api-docs-view/index.tsx

```diff
@@ -2,27 +2,98 @@
 import SidebarLayout from 'layouts/sidebar-layout'
 // Local
 import { OpenApiDocsMobileMenuLevels } from './components'
+import { DevCodeBlock, OperationSections } from './components'
 // Types
-import type { OpenApiDocsViewProps } from './types'
+import type { OpenApiDocsViewProps, OperationProps } from './types'
 
 /**
  * Placeholder for a revised OpenAPI docs view.
  */
-function OpenApiDocsView({ productData, ...restProps }: OpenApiDocsViewProps) {
+function OpenApiDocsView({
+	productData,
+	operationGroups,
+	...restProps
+}: OpenApiDocsViewProps) {
 	return (
 		<SidebarLayout
 			sidebarSlot={
-				<div style={{ border: '1px solid magenta' }}>
-					PLACEHOLDER for sidebar contents
+				<div style={{ border: '1px solid magenta', overflow: 'hidden' }}>
+					{/**
+					 *
+					 * Placeholder for sidebar (and mobile nav?) contents
+					 *
+					 **/}
+					{operationGroups.map((group) => {
+						return (
+							<>
+								<div>{group.heading}</div>
+								<ul key={group.heading}>
+									{group.items.map((o: OperationProps) => {
+										return (
+											<li key={o.operationId}>
+												<a href={`#${o.slug}`}>{o.operationId}</a>
+											</li>
+										)
+									})}
+								</ul>
+							</>
+						)
+					})}
 				</div>
 			}
 			mobileMenuSlot={<OpenApiDocsMobileMenuLevels productData={productData} />}
 		>
+			{/**
+			 *
+			 * Placeholder for overview
+			 *
+			 **/}
```

Comment:
💡 would make it even easier to avoid conflicts if these were emptively split out into their own components. Not necessary, just an idea.

Reply:
Oooh yeah great call! I'll do this now 👍

Reply:
✅ Done! Split out all the placeholders into their own folders in `src/views/open-api-docs-view/components` 👍